src/services/search-builder.js

```diff
@@ -54,6 +54,20 @@ function SearchBuilder(model, opts, params, fieldNamesRequested) {
     }
   }
 
+  function getStringExtendedCondition(attributes, value, column) {
+    let condition = {};
+
+    if (attributes && isUUID(DataTypes, attributes.type)) {
+      if (value.match(REGEX_UUID)) {
+        condition = opts.sequelize.where(column, '=', value);
+      }
+    } else {
+      condition = opts.sequelize.where(lowerIfNecessary(column), ' LIKE ', lowerIfNecessary(`%${value}%`));
+    }
+
+    return condition;
+  }
```

Comment:
[Suggestion, ignorable] I guess this function could be simplified, but feel free to keep it like this:

- We could avoid `let`
- A `else` is rarely required.
- Maybe we could change `{}` by `null`, it will simplify code `!_.isEmpty(condition)`.

```suggestion
  function getStringExtendedCondition(attributes, value, column) {
    if (attributes && isUUID(DataTypes, attributes.type)) {
      if (!value.match(REGEX_UUID)) return null;
      return opts.sequelize.where(column, '=', value);
    }
    return opts.sequelize.where(lowerIfNecessary(column), ' LIKE ', lowerIfNecessary(`%${value}%`));
  }
```

But, again, if you find it more readable the way you write it, keep it like this!

Reply:
Thanks a lot!
I reworked this part ;)